opendebates/tests/test_navigation.py

```diff
@@ -52,9 +52,10 @@ def test_search_term_is_sticky(self):
 
     def test_search_term_inserts_into_sort_form(self):
         rsp = self.client.get(self.search_url + "?q=something+ducks")
-        form_html = (r'\<form action="{search}" method="GET" '
-                     r'class="form-inline"\>\W+'
-                     r'\<input type="hidden" name="q" '
-                     r'value="something ducks"\>'.format(
+        form_html = ('<form action="{search}" method="GET" '
+                     'class="form-inline">'
+                     r'\W+'
```

Comment:
This seems to be mixing regular strings with raw strings across the lines - not sure what Python will do with it but can we just consistently use raw strings here?

Reply:
Sure